deploy/README.md

```diff
@@ -92,7 +92,75 @@ If you need to restart the raster tileserver:
 
 ```bash
 # restart the service
-sudo service restart terracotta
+sudo service terracotta restart
```

Comment:
Isn't it actually like the original? `service` is the command, `restart` the subcommand, and `terracotta` the argument for the service name.
Also, I think `systemctl` is the most commonly use command nowadays?

Reply:
Yeah, I'm actually using `systemctl` - `service` is still on the fingertips somehow.